examples/layers/examples/nonce_layer.rs

```diff
@@ -40,7 +40,7 @@ async fn main() -> Result<()> {
         // includes the `NonceManagerLayer`.
         .layer(NonceManagerLayer)
         .signer(EthereumSigner::from(signer))
-        .on_client(RpcClient::new_http(rpc_url));
+        .on_reqwest_http(rpc_url)?;
```

Comment:
@prestwich considering we default to the `reqwest` crate in the `alloy` prelude, would it be possible to create an `on_http` alias? When the `hyper` crate is enabled `on_http` would use the `hyper` implementation rather than the `reqwest` implementation?

Main point is to not bother the end user with implementation details

Reply:
I think this is superseded by the conversation in regards to `Provider::builder()` and we can handle it as part of that

Reply:
yeah, I wouldn't mind making "http" mean reqwest everywhere, and then, having hyper in the name for all methods associated with hyper transport